client/src/pages/BodyParts.js

```diff
@@ -1,9 +1,8 @@
 import { Link } from 'react-router-dom';
 import styled from 'styled-components';
-import { ReactComponent as BodyParts_overview } from '../assets/Bodypart_overview.svg';
+import { ReactComponent as BodyParts_overview } from '../assets/bodyPartOverview.svg';
```

Comment:
```suggestion
import { ReactComponent as BodyPartsOverview } from '../assets/bodyPartOverview.svg';
```

The rest of your app uses camel case, why do a mix of camel and snake here?